mantidimaging/gui/test/gui_system_reconstruction_test.py

```diff
@@ -44,14 +44,35 @@ def tearDown(self) -> None:
         self.assertFalse(self.main_window.isVisible())
 
     def test_correlate(self):
+        expected_initial_cor = self.recon_window.rotation_centre
+        expected_initial_tilt = self.recon_window.tilt
+
+        expected_final_cor = 66.0
+        expected_final_tilt = 1.070368
+
         for _ in range(5):
             QTest.mouseClick(self.recon_window.correlateBtn, Qt.MouseButton.LeftButton)
             QTest.qWait(SHORT_DELAY)
             wait_until(lambda: self.recon_window.correlateBtn.isEnabled())
             wait_until(lambda: len(self.recon_window.presenter.async_tracker) == 0)
 
+        final_cor_value = self.recon_window.rotation_centre
+        final_tilt_value = self.recon_window.tilt
+
+        assert final_cor_value != expected_initial_cor
+        assert final_tilt_value != expected_initial_tilt
+
+        assert final_cor_value == expected_final_cor
+        assert final_tilt_value == expected_final_tilt
```

Comment:
All of the asserts should be `self.assertEqual()` as that gives better messages.